bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -142,7 +143,6 @@ private func dummy() {
     _ = NSLocalizedString("Embed block previews are coming soon", comment: "")
     _ = NSLocalizedString("Embed caption. %s", comment: "translators: accessibility text. %s: Embed caption.")
     _ = NSLocalizedString("Embed caption. Empty", comment: "translators: accessibility text. Empty Embed caption.")
-    _ = NSLocalizedString("Embed link", comment: "")
```

Comment:
- Removal
- No code reference

This string is not being used anywhere so we can safely remove it, probably the string changed.